### ISSUE TITLE
Failed to activate the comment-anchors package

### ISSUE THREAD

**Issue by oneeyedman** [callodacity/comment-anchors#4]
I simply fire the shortcut and BAM! :(
I have a custom regexp: ^\s*\/{2} >(.+)

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [comment-anchors](https://github.com/callodacity/comment-anchors) package 0.4.6

### Stack Trace

Failed to activate the comment-anchors package

```
At Cannot read property 'getGrammar' of undefined

TypeError: Cannot read property 'getGrammar' of undefined
    at MySelectListView.module.exports.MySelectListView.getItems (/packages/comment-anchors/lib/comment-anchors-view.coffee:83:21)
    at MySelectListView.module.exports.MySelectListView.initialize (/packages/comment-anchors/lib/comment-anchors-view.coffee:17:16)
    at /packages/jquery-snippets/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
    at /packages/jquery-snippets/node_modules/atom-space-pen-views/lib/select-list-view.js:18:51)
    at /packages/comment-anchors/lib/comment-anchors-view.coffee:14:3)
    at Object.activate (/packages/comment-anchors/lib/comment-anchors.coffee:13:31)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:968:1
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:246:1)
    at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:1)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:1)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:1)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:1)
```

### Commands

```
     -8:16.2.0 core:paste (input.hidden-input)
     -8:15.8.0 editor:move-to-first-character-of-line (input.hidden-input)
  5x -8:15.6.0 core:move-right (input.hidden-input)
     -8:14.5.0 core:move-left (input.hidden-input)
  2x -8:09.2.0 core:save (input.hidden-input)
     -6:47.6.0 core:save-as (ul.list-inline.tab-bar.inset-panel)
     -6:42.2.0 settings-view:open (ul.list-inline.tab-bar.inset-panel)
     -6:05.4.0 core:copy (input.hidden-input)
     -5:35.9.0 comment-anchors:toggle (div.tool-panel.tree-view)
     -3:56.3.0 atom-alignment:alignMultiple (input.hidden-input)
  5x -3:54 comment-anchors:toggle (input.hidden-input)
     -2:12.5.0 core:move-left (input.hidden-input)
     -2:10.3.0 core:save (input.hidden-input)
     -0:41.8.0 core:copy (input.hidden-input)
     -0:21.4.0 core:paste (input.hidden-input)
     -0:09.5.0 comment-anchors:toggle (div.tool-panel.tree-view)
```

### Non-Core Packages

```
activate-power-mode 1.3.0 
apple-wwdc-2016-syntax 0.6.0 
atom-alignment 0.12.1 
atom-beautify 0.29.17 
atom-css-comb 3.1.0 
atom-jade 0.3.0 
atom-material-syntax 1.0.2 
atom-material-syntax-dark 0.2.7 
atom-material-syntax-light 0.4.6 
atom-material-ui 2.0.3 
atom-panda-syntax 0.18.0 
comment-anchors 0.4.6 
comment-timestamp 0.3.0 
docblockr 0.9.1 
drunken-php 0.2.3 
duplicate-and-comment 0.2.5 
emmet 2.4.3 
emmet-snippets-compatibility 1.0.6 
firewatch-syntax 1.1.1 
git-time-machine 1.5.6 
html-include-resource 1.3.0 
insert-random-comments 0.1.0 
jquery-snippets 11.0.0 
jslint 1.5.1 
language-pug 0.0.21 
linter 1.11.23 
linter-stylelint 3.9.0 
markdown-pdf 1.5.0 
markdown-toc 0.4.2 
markdown-writer 2.6.4 
merge-conflicts 1.4.4 
minimap 4.26.8 
php-twig 4.0.0 
pigments 0.39.0 
pretty-json 1.6.1 
project-manager 3.3.3 
random 1.3.0 
seti-syntax 1.1.2 
seti-ui 1.6.1 
sort-lines 0.14.0 
split-diff 1.2.0 
sync-settings 0.8.1 
toggle-markdown-task 0.6.0 
```

**Comment by callodacity** [comment]
Ah, good catch! That's not your custom regexp, that occurs when you don't have any text editors that are currently active (I can reproduce it if I select the side-menu and then fire the shortcut).

I'll fix that up 👍

**Comment by callodacity** [comment]
@oneeyedman should be fixed in [`0.4.7`](https://github.com/callodacity/comment-anchors/releases/tag/v0.4.7)

**Comment by oneeyedman** [comment]
Hi! I have an update. It was "Atom Material" UI theme's fault (when applied this theme you only see the search bar). I've changed to another UI Theme and it works right
